aio/dune

```diff
@@ -0,0 +1,5 @@
+(executables
+ (names echo)
+ (libraries unix)
+ (modules aio echo)
+ (modes byte exe))
```

Comment:
We don't need the bytecode mode. Observe that `ocamlbuild` command only builds the native version.

Reply:
Saw that later in the makefile.